client/blocks/eligibility-warnings/datacenter-picker.tsx

```diff
@@ -0,0 +1,155 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import styled from '@emotion/styled';
```

Comment:
We're not using Emotion anywhere else in `client/blocks/eligibility-warnings` so I'd be reluctant to add it for this component without a compelling reason. My guess this is what's adding the few extra KBs to the async bundles.

Reply:
> We're not using Emotion anywhere else in `client/blocks/eligibility-warnings` so I'd be reluctant to add it for this component without a compelling reason.

My understanding is that we want to use CSS objects for all new components, and when we have an opportunity to refactor existing components.

Reply:
Interesting! That's not my impression at all. Seemed like there have been a number of conversations like this p1666216809948199-slack-C02A41GCS where folks have been questioning its usefulness.